conan/tools/sbom/cyclonedx.py

```diff
@@ -3,6 +3,28 @@
 def cyclonedx_1_4(graph, name=None, add_build=False, add_tests=False, **kwargs):
     """
     (Experimental) Generate cyclone 1.4 sbom with json format
```

Comment:
I think I would change this too, to:

`(Experimental) Generate a CycloneDX 1.4 SBOM in JSON format.`